Description: wrong version number in CMakeLists.txt. 
 Patch only for 0.2.0 release
Author: Guillaume PASERO <guillaume.pasero@c-s.fr>

--- a/CMakeLists.txt	Thu Sep 18 14:33:49 2014 +0200
+++ b/CMakeLists.txt	Thu Sep 18 14:34:55 2014 +0200
@@ -3,7 +3,7 @@
 cmake_minimum_required(VERSION 2.6)
 
 set(Ice_VERSION_MAJOR "0")
-set(Ice_VERSION_MINOR "1")
+set(Ice_VERSION_MINOR "2")
 set(Ice_VERSION_PATCH "0")
 
 set(CMAKE_MODULE_PATH
